src/figuresClasses.ts

```diff
@@ -1,19 +1,69 @@
-export interface Figure {
+type Color = 'red' | 'green' | 'blue';
+type Shape = 'triangle' | 'circle' | 'rectangle';
 
+export interface Figure {
+  shape: Shape,
+  color: Color,
+  getArea(): number,
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  shape: Shape= 'triangle';
```

Comment:
Let's create the enum `Shape` with the shape values, so we could restrict those values that are being passed to the shape property